### PR TITLE
Added Translation process. Configured language not recognized

### LINKED ISSUE #101: Translation process
## Description

- Add process including translation. 
- Configure server

### PR DESCRIPTION
**Pull request by huard**
## Overview

This PR fixes #101

Changes:

* Added Translation process
* Tried to configure server, but does not seem to work.

## Related Issue / Discussion

https://github.com/bird-house/birdy/pull/164

## Additional Information

Links to other issues or sources.

### CODE REVIEW

File: emu/processes/__init__.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 from .wps_ncmeta import NCMeta
 from .wps_nonpyid import NonPyID
 from .wps_dry_run import SimpleDryRun
+from .wps_translation import Translation
```
Comment (cehbrecht) [inline emu/processes/__init__.py:19]: the `wps_translation` module is missing in this PR.

Comment (huard) [reply emu/processes/__init__.py:19]: Oups ! Added.

**Comment by cehbrecht** [comment]
get capabilities request ... :
http://localhost:5000/wps?request=GetCapabilities&service=WPS

... only shows en-US language:
```
<wps:Languages>
    <wps:Default>
      <ows:Language>en-US</ows:Language>
    </wps:Default>
    <wps:Supported>
      <ows:Language>en-US</ows:Language>
    </wps:Supported>
  </wps:Languages>
```

But the pywps 4.2.4 version should have language support:
https://github.com/geopython/pywps/pull/510

**Comment by cehbrecht** [comment]
@huard tests are failing. `translation` process needs to be added the `test_wps_caps.py` test.

File: emu/default.cfg [hunk 2]
```diff
@@ -13,6 +13,7 @@ allowedinputpaths = /
 maxsingleinputsize = 200mb
 maxprocesses = 10
 parallelprocesses = 2
+languages = en-US,fr-CA
```
Comment (cehbrecht) [inline emu/default.cfg:16]: config parameter is `language` ... no `s`.

Comment (cehbrecht) [reply emu/default.cfg:16]: made a local change. works with fixed parameter name.

Comment (huard) [reply emu/default.cfg:16]: Ahhh ! Thanks !

**Comment by cehbrecht** [comment]
Translation is not shown:
http://localhost:5000/wps?request=DescribeProcess&service=WPS&version=1.0.0&identifier=translation&language=fr-CA

... title in English, not French:
```
<DataInputs>
      <ows:Title>Input1 number</ows:Title>
```

**Comment by huard** [comment]
There was a mismatch between process and config. Should work now.

**Comment by huard** [comment]
Want to add  a German translation for your own demos ?

**Comment by cehbrecht** [comment]
> Want to add a German translation for your own demos ?

@huard sure. I add it and make the merge.

**Comment by cehbrecht** [comment]
@huard I have added support for de-DE and fixed the tests and `test.cfg` config.

File: emu/processes/wps_translation.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import logging
+LOGGER = logging.getLogger("PYWPS")
+
+
+class Translation(Process):
+    def __init__(self):
+        inputs = [
+            LiteralInput('input1', 'Input1 number',
+                         default='100', data_type='integer',
+                         translations={"fr_ca": {"title": "Entrée #1", "abstract": "Entier #1"}}),
```
[the remark this answers is not in the document]
Comment (cehbrecht) [reply emu/processes/wps_translation.py]: done.

**Pull request merged**